ext.php

```diff
@@ -17,6 +17,6 @@ class ext extends \phpbb\extension\base
 	public function is_enableable()
 	{
 		$config = $this->container->get('config');
-		return version_compare($config['version'], '3.1.4', '>=') && version_compare(PHP_VERSION, '5.4.*', '>'));
+		return (version_compare($config['version'], '3.1.4', '>=') && (version_compare(PHP_VERSION, '5.4.*', '>')));
```

Comment:
The outer parenthesis isn't required and I kind of prefer it left off.

```
return version_compare($config['version'], '3.1.4', '>=') && version_compare(PHP_VERSION, '5.4.*', '>');
```

But you are right, there is an extra parenthesis we both missed in that first PR.